enhancements/baremetal/baremetal-keepalived-unicast.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Baremetal-Keepalived-Unicast
+authors:
+  - "@bcrochet"
+  - "@yboaron"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-21
+last-updated: 2020-02-21
+status: implementable
+---
+
+# baremetal-keepalived-unicast
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The Baremetal platform uses VIPs (Virtual IP) and [Keepalived](https://www.keepalived.org/) to 
+support high availability for API and default Ingress.
+A VIP is used to provide failover of the service across the relevant nodes (including the bootstrap instance).
+
+Two VIPs are supported:
+* api-vip
+* ingress-vip
+
+The `api-vip` first resides on the bootstrap instance and it moves to one of the control plane nodes, but only after the bootstrap process has completed and the bootstrap instance is stopped. This ensures that the API on the control plane nodes is fully functional before the api-vip moves.
+The `ingress-vip` resides on one of the nodes that runs a router pod.
+
+In current implementation, Keepalived is configured to run [VRRP management protocol](https://tools.ietf.org/html/rfc3768#section-3) of the VIP addresses in a multicast mode, but since not all environments have multicast available, this proposal seeks to remedy that.
+
```

Comment:
A helpful link to add here would be https://github.com/openshift/installer/blob/master/docs/design/baremetal/networking-infrastructure.md

That doc covers the overview you provide here in some more detail

Reply:
Done